### PR TITLE
Swap from isomorphic-fetch to fetch-everywhere. RE: #192

### LINKED ISSUE #192: ReferenceError: self is not defined
Hey, thanks for the amazing component!

I've been making a few experiments with it and I ended up facing the following:

```sh
self is not defined
ReferenceError: self is not defined
    at Object.<anonymous> (/Users/mabrasil/Documents/Github/beakyn/docs/final2/node_modules/react-lincoln/Lincoln.js:19:58659)
    at t (/Users/mabrasil/Documents/Github/beakyn/docs/final2/node_modules/react-lincoln/Lincoln.js:1:177)
    at Object.module.exports.Object.defineProperty.value (/Users/mabrasil/Documents/Github/beakyn/docs/final2/node_modules/react-lincoln/Lincoln.js:19:173470)
    at t (/Users/mabrasil/Documents/Github/beakyn/docs/final2/node_modules/react-lincoln/Lincoln.js:1:177)
    at Object.module.exports.Object.defineProperty.value (/Users/mabrasil/Documents/Github/beakyn/docs/final2/node_modules/react-lincoln/Lincoln.js:19:166713)
    at t (/Users/mabrasil/Documents/Github/beakyn/docs/final2/node_modules/react-lincoln/Lincoln.js:1:177)
    at Object.module.exports.Object.defineProperty.value (/Users/mabrasil/Documents/Github/beakyn/docs/final2/node_modules/react-lincoln/Lincoln.js:19:101177)
    at t (/Users/mabrasil/Documents/Github/beakyn/docs/final2/node_modules/react-lincoln/Lincoln.js:1:177)
    at Object.<anonymous> (/Users/mabrasil/Documents/Github/beakyn/docs/final2/node_modules/react-lincoln/Lincoln.js:49:115114)
    at t (/Users/mabrasil/Documents/Github/beakyn/docs/final2/node_modules/react-lincoln/Lincoln.js:1:177)
```

There're many [other](https://github.com/markdalgleish/static-site-generator-webpack-plugin/issues/79) [related](https://github.com/webpack-contrib/style-loader/issues/177) [issues](https://github.com/webpack-contrib/extract-text-webpack-plugin/issues/453) out there and it seems to be a problem with `webpack`–and not with `Lincoln` itself–but none of the solutions seemed to be related.

[One of the comments](https://github.com/webpack-contrib/style-loader/issues/177#issuecomment-283805278) got my attention:

> The problem here is that global.self is only defined in Browsers and Web Workers. Unit tests running in NodeJs environment won't have access to global.self.

I think this can be kind of a clue, since I'm doing server-side rendering with [next.js](https://github.com/zeit/next.js) but I still can't figure it out.

Any thoughts?

### PR DESCRIPTION
**Pull request by brendo**
Could perhaps resolve #192!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,7 +6,13 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 
 ## [Unreleased]
 
+### Changed
+
+- Various styling enhancements to typography and colours.
```
Comment (nfour) [inline CHANGELOG.md:11]: Why is this here in this MR?

**Review by nfour — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (brendo) [reply CHANGELOG.md:11]: I forgot to add this stuff to the changelog last time in #188, so just doing it this time around :)

**Comment by nfour** [comment]
Merged because there is more than one issue with isomorphic-fetch

**Comment by nfour** [comment]
Ah crap, @brendo this library uses `isomorphic-fetch` under the hood anyway?
